js/mainApiMgr.js

```diff
@@ -82,6 +83,11 @@ electron.ipcMain.on(apiName, (event, arg) => {
     }
 
     if (arg.cmd === apiCmds.activate && typeof arg.windowName === 'string') {
```

Comment:
Not good having all these ```if``` blocks. Should probably switch to a switch-case block (no pun).

Reply:
Update in this PR -> https://github.com/symphonyoss/SymphonyElectron/pull/292